indico/modules/events/editing/client/js/management/editable_type/EditableList.jsx

```diff
@@ -325,31 +388,44 @@ function EditableListDisplay({initialEditableList, codePresent, editableType, ev
                 rowCount={sortedList.length}
                 scrollTop={scrollTop}
                 rowGetter={({index}) => sortedList[index]}
+                rowStyle={({index}) =>
+                  index !== -1 && !filteredSet.has(sortedList[index].id)
+                    ? {backgroundColor: Palette.pastelGray, opacity: '60%'}
+                    : {}
+                }
               >
                 <Column
                   disableSort
                   dataKey="checkbox"
                   width={30}
                   headerRenderer={() => (
                     <Checkbox
-                      indeterminate={checked.length > 0 && checked.length < editables.length}
-                      checked={checked.length === editables.length}
-                      onChange={(e, data) => toggleSelectAll(data.checked)}
-                      disabled={!!activeRequest}
-                    />
-                  )}
-                  cellRenderer={({rowIndex}) => (
-                    <Checkbox
-                      disabled={!sortedList[rowIndex].editable || !!activeRequest}
+                      indeterminate={
+                        checked.length > 0 &&
+                        checked.length < [...filteredSet].filter(x => contribIdSet.has(x)).length
+                      }
                       checked={
-                        sortedList[rowIndex].editable
-                          ? checked.includes(sortedList[rowIndex].editable.id)
-                          : false
+                        checked.length > 0 &&
+                        checked.length === [...filteredSet].filter(x => contribIdSet.has(x)).length
                       }
-                      onChange={(e, data) => toggleSelectRow(data.index)}
-                      index={rowIndex}
+                      onChange={(e, data) => toggleSelectAll(data.checked)}
+                      disabled={!!activeRequest}
                     />
                   )}
+                  cellRenderer={({rowIndex}) =>
+                    sortedList[rowIndex].editable && (
+                      <Checkbox
+                        disabled={!!activeRequest || !filteredSet.has(sortedList[rowIndex].id)}
+                        checked={
+                          sortedList[rowIndex].editable
+                            ? checked.includes(sortedList[rowIndex].editable.id)
```

Comment:
We can use `checkedSet.has(sortedList[rowIndex].editable.id)` here as well... it was already using `includes` before but it makes sense to use the Set here..

Same for the other place where we currently use `checked.includes`.